javascript/javascript1/week1/homework-Ali-Nematollahi/Age-ify.js

```diff
@@ -0,0 +1,12 @@
+console.log('====== Age-ify ======');
+
+
+let yearOfBirth = 1989;
+let yearFuture = 2050;
+
+let age = yearFuture - yearOfBirth;
+
+console.log('You will be' , age , 'years old in' , yearFuture);
```

Comment:
This one could be logged a bit better if you had written it like this:
```suggestion
console.log("You will be " + age + "' years old in " + yearFuture);
```
The result now is 
```
"You will be" 61 "years old in" 2050
```
compared to
```
"You will be 61 years old in 2050"
```